frontend/src/components/util.tsx

```diff
@@ -191,30 +195,10 @@ export const ActionWithDialog = ({
     | "ghost"
     | null
     | undefined;
-  /**
-   * For some ops (Delete), force confirm dialog
-   * even if disabled.
-   */
-  forceConfirmDialog?: boolean;
 }) => {
-  const disable_confirm_dialog =
-    useRead("GetCoreInfo", {}).data?.disable_confirm_dialog ?? false;
```

Comment:
This looks like you remove the KOMODO_DISABLE_CONFIRM_DIALOG feature with this, or am I missing somthing?